addons/event_crm/views/crm_lead_views.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="crm_lead_view_form_inherited" model="ir.ui.view">
+        <field name="name">crm.lead.form.inherited</field>
+        <field name="model">crm.lead</field>
+        <field name="inherit_id" ref="crm.crm_lead_view_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//div[@name='button_box']" position="inside">
+                <button name="action_get_attendee" type="object" class="oe_stat_button" icon="fa-users"
+                    attrs="{'invisible': [('attendees_count', '=', 0)]}">
+                    <div class="o_stat_info">
+                        <field name="attendees_count"/>
+                        <span class="o_stat_text" attrs="{'invisible': [('attendees_count', '&gt;', 1)]}"> Attendee</span>
+                        <span class="o_stat_text" attrs="{'invisible': [('attendees_count', '&lt;=', 1)]}"> Attendees</span>
```

Comment:
registration instead of attendee -> and use `Registrations` (or `Registration(s)`)

Reply:
done